sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1214,34 +1213,35 @@ def _from_generated(cls, generated):
         return None
 
 
-class DelimitedTextConfiguration(GenDelimitedTextConfiguration):
-    """Defines the input or output delimited (CSV) serialization for a blob quick query request.
+class DelimitedJSON(object):
```

Comment:
There's an existing comment about removing this class, service will default it to '\n' even nothing is specified, how do you feel?